tests/enumerror/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -33,14 +33,12 @@ function isActive(
 ): boolean {
   return ad.state === \"ACTIVE\";
 }
-
 isActive({ state: \"PAUSE\" });
 
 var MyStates = { PAUSED: \"PAUSED\", ACTIVE: \"ACTIVE\", DELETED: \"DELETED\" };
 function isActive2(ad: { state: $Keys<typeof MyStates> }): boolean {
   return ad.state === MyStates.ACTIVE;
 }
-
```

Comment:
I can go on for the entire file but it's basically 50/50 correct and incorrect. I think we should not have the `+ 1` that doesn't make any sense and try to figure out why the newly incorrect cases here happen and fix them for real.